File: BWOnlineEducationiOS/BWOnlineEducationiOS/MVC/Base/BWBaseViewController.m
//
//  BWBaseViewController.m
//  BWOnlineEducationiOS
//
//  Created by BobWong on 2017/6/6.
//  Copyright © 2017年 Bob Wong Studio. All rights reserved.
//

#import "BWBaseViewController.h"

@interface BWBaseViewController ()

@end

@implementation BWBaseViewController

#pragma mark - Life Cycle

- (void)viewDidLoad {
    [super viewDidLoad];
}

- (void)dealloc {
    NSLog(@"dealloc %@", NSStringFromClass([self class]));
}

@end

File: BWOnlineEducationiOSInSwift/BWOnlineEducationiOSInSwift/MVC/Base/BWBaseViewController.swift
//
//  BWBaseViewController.swift
//  BWOnlineEducationiOSInSwift
//
//  Created by BobWong on 2017/6/19.
//  Copyright © 2017年 BobWongStudio. All rights reserved.
//

import UIKit

class BWBaseViewController: UIViewController {

    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.view.backgroundColor = UIColor.white
    }
    
    deinit {
        print("deinit \(String(describing: self))")
    }

}
